axum-core/src/macros.rs

```diff
@@ -49,12 +49,15 @@ macro_rules! __define_rejection {
 
         impl $crate::response::IntoResponse for $name {
             fn into_response(self) -> $crate::response::Response {
+                let status = self.status();
+                let body_text = self.body_text();
```

Comment:
How is using `self.body_text()` better than using `$body`? I think the latter is always a string literal. It should probably be using the `literal` (`lit`?) macro fragment specifier instead of `expr`.

Reply:
I think I subconsciously aimed towards having the same `into_response()` content for both cases, but you're right that `$body` is better, because `body_text()` turns the `&'static str` into a `String` which isn't what we want here. I've reverted that part.